backend/app/infrastructure/__init__.py

```diff
@@ -56,20 +60,62 @@ def __getattribute__(self, name):
         return attr
 
 
+def lookup_max_time():
+    assert max_time is not None
+    return max_time
+
+
+def is_graph_error_permanent(error):
+    # TODO this must differentiate between different sources reasons for
+    #      failure
+    # Figure out where exactly is string about reorgs is hidden
+    try:
+        print(f"got TransportQueryError.query_id: {error.query_id}")
+        print(f"got TransportQueryError.errors: {error.errors}")
+        print(f"got TransportQueryError.data: {error.data}")
+        print(f"got TransportQueryError.extensions: {error.extensions}")
+    except:  # noqa: B001, E722
+        pass
+    return False
+
+
+class GQLWithRetryBackoff(Client):
+    """
+    It would be nice to use something out of the box from GQL
```

Comment:
Maybe let's change it to more docstr-related style?
For instance:

```
A retry wrapper for async transports. It overrides execute() method to handle TransportQueryError and uses @backoff decorator to make it retryable for given period of time.
```

Reply:
+1